app/settings.py

```diff
@@ -28,3 +28,8 @@
 # Proxy settings
 FORWARED_ALLOW_IPS = os.getenv('FORWARED_ALLOW_IPS', '*')
 FORWARDED_PROTO_HEADER_NAME = os.getenv('FORWARDED_PROTO_HEADER_NAME', 'X-Forwarded-Proto')
+WSGI_WORKERS = int(os.getenv('WSGI_WORKERS', '2'))
+if WSGI_WORKERS <= 0:
+    from multiprocessing import cpu_count
+    WSGI_WORKERS = (cpu_count() * 2) + 1
+WSGI_TIMEOUT = int(os.getenv('WSGI_TIMEOUT', '30'))
```

Comment:
Shouldn't we aim for a lower timeout in general? Setting 30s as default seems high...